app/src/androidTest/java/com/example/concordia_campus_guide/EspressoHelpers.java

```diff
@@ -79,4 +90,79 @@ public static void cancelNotificationIfExists(){
             //ignore maybe the popup was cancelled by other test
         }
     }
+
+    //used for screenshot testing
+    public static String BitmapToBase64String(Bitmap bitmap){
+        ByteArrayOutputStream byteArrayOutputStream = new ByteArrayOutputStream();
+        bitmap.compress(Bitmap.CompressFormat.PNG, 100, byteArrayOutputStream);
+        byte[] byteArray = byteArrayOutputStream .toByteArray();
+
+        String encoded = Base64.encodeToString(byteArray, Base64.DEFAULT);
+        return encoded;
+    }
+
+    public static Bitmap importScreenshot(String path){
+        String fileString = null;
+
+        try
+        {
+            AssetManager am = InstrumentationRegistry.getInstrumentation().getContext().getAssets();
+            InputStream inputStream = am.open(path);
+            Scanner s = new Scanner(inputStream).useDelimiter("\\A");
+            fileString =s.hasNext() ? s.next() : "";
+        }
+        catch (IOException e)
+        {
+            e.printStackTrace();
+        }
+
+        byte[] decodedString = Base64.decode(fileString, Base64.DEFAULT);
+        return BitmapFactory.decodeByteArray(decodedString, 0, decodedString.length);
+    }
+
+    public static boolean imagesAreEqualExceptInfoCardHeight(Bitmap bm1, Bitmap bm2){
+        if (bm1.getHeight() != bm2.getHeight() || bm1.getWidth() != bm2.getWidth()){
+            return false;
+        }
+
+        Bitmap bmp = bm1.copy(bm1.getConfig(), true);
+        int threshold = 10;
+        boolean isSame = true;
+
+        //use coordinates to reduce flakiness of screenshot testing
+        int x1 = 470;
+        int x2 = 618;
+        int y1 = 989;
+        int y2 = 1139;
```

Comment:
Do you think we can use more representative names?

Reply:
these are x,y coordinates on the screenshot to make the triangle, do you have any ideas for more representative names?